tests/integration/schema/updates/add/field/kind/foreign_object_test.go

```diff
@@ -952,3 +894,159 @@ func TestSchemaUpdatesAddFieldKindForeignObject_ObjectKindAndSchemaMismatch(t *t
 	}
 	testUtils.ExecuteTestCase(t, test)
 }
+
+func TestSchemaUpdatesAddFieldKindForeignObject_MissingPrimaryIDField(t *testing.T) {
+	key1 := "bae-decf6467-4c7c-50d7-b09d-0a7097ef6bad"
+
+	test := testUtils.TestCase{
+		Description: "Test schema update, add field with kind foreign object (16), with auto primary ID field creation",
+		Actions: []any{
+			testUtils.SchemaUpdate{
+				Schema: `
+					type Users {
+						name: String
+					}
+				`,
```

Comment:
suggestion: Same as above. It would be nice to have a test for two separate schemas. In this case it would help show that the `relation_id` is set on both sides.